bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestPrecompute.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace ElectionGuard.Encryption.Tests
+{
+    [TestFixture]
+    class TestPrecompute
+    {
+        [Test]
+        public void Test_Precompute_Status_NoStarted()
+        {
+            Precompute precompute = new Precompute();
+
+            var status = precompute.GetStatus();
+
+            Assert.That(status.CurrentState == PrecomputeState.NotStarted);

```

Comment:
I generally prefer `Assert.AreEqual` instead of `Assert.That` because when the latter fails you'll get a less helpful message like `Expected: True But was:  False` and when the former fails you get something like `Expected: 1 But was:  2`.

Reply:
Agreed.  Fixed the asserts.